### DISCUSSION TITLE
How to run .py files..

### DISCUSSION THREAD

**Discussion by Aleem28** [community/community#125443] [Programming Help] [answered]
### Select Topic Area

Question

### Body

Hello guys, iam new in github and i have no idea of how to use these files. I want to download some files in rapidgator but the time and speed restriction prevent me to do so.
Can anyone send me video explaing step by step how to install rapidgator downloder or How to use python script. I found few links for rapidgator but all files are in **.py** format I don't know how to use or run. Has no clue on this file execution. Can anyone explain please.

Thankyou

**Comment by Mahmud-cse** [comment, marked as answer]
`import requests

url = 'your_rapidgator_link'
output_file = 'downloaded_file.zip'

response = requests.get(url)
with open(output_file, 'wb') as file:
    file.write(response.content)

print(f"Downloaded {output_file}")
`
save the code as rapidgator_downloader.py and run the script. It should work.

**Reply by Aleem28** [reply]
Shukriya bro. But frankly I don't know how to use. Can you help me with any video if available.

**Comment by cehfisher** [comment]
Thanks for posting in the GitHub Community, @Aleem28! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them. I've asked @queenofcorgis to move this issue into the correct category for you. Thank you!
